clients/common/rocblas_gentest.py

```diff
@@ -202,38 +214,25 @@ def setdefaults(test):
                             'dotc_strided_batched', 'rot_strided_batched',
                             'rotm_strided_batched', 'iamax_strided_batched',
                             'iamin_strided_batched', 'axpy_strided_batched'):
-        if all([x in test for x in ('N', 'incx', 'stride_scale')]):
-            ldx = int(test['N'] * abs(test['incx']) * test['stride_scale'])
-            test.setdefault('stride_x', ldx)
-        if all([x in test for x in ('N', 'incy', 'stride_scale')]):
-            ldy = int(test['N'] * abs(test['incy']) * test['stride_scale'])
-            test.setdefault('stride_y', ldy)
+        setkey_product(test, 'stride_x', ['N', 'incx', 'stride_scale'])
+        setkey_product(test, 'stride_y', ['N', 'incy', 'stride_scale'])
         # we are using stride_c for param in rotm
         if all([x in test for x in ('stride_scale')]):
             test.setdefault('stride_c', int(test['stride_scale']) * 5)
 
     elif test['function'] in ('trmv_strided_batched'):
-        if all([x in test for x in ('M', 'incx', 'stride_scale')]):
-            ldx = int(test['M'] * abs(test['incx']) * test['stride_scale'])
-            test.setdefault('stride_x', ldx)
-        if all([x in test for x in ('M', 'lda', 'stride_scale')]):
-            ldM = int(test['M'] * test['lda'] * test['stride_scale'])
-            test.setdefault('stride_a', ldM)
-    elif test['function'] in ('gemv_strided_batched', 'ger_strided_batched', 'trsv_strided_batched'):
+        setkey_product(test, 'stride_x', ['M', 'incx', 'stride_scale'])
+        setkey_product(test, 'stride_a', ['M', 'lda', 'stride_scale'])
+
+    elif test['function'] in ('gemv_strided_batched', 'gbmv_strided_batched', 'ger_strided_batched', 'trsv_strided_batched'):
```

Comment:
>   elif test['function'] in ('gemv_strided_batched', 'gbmv_strided_batched', 'ger_strided_batched', 'trsv_strided_batched'):

These lines are getting long. I've run the current `rocblas_gentest.py` through `flake8`, and made changes in PR #907 to avoid long lines.  There might be merge conflicts between these two changes.

Except for the handling of known bugs, the other changes in `rocblas_gentest.py` were simply to split long lines into two. Python allows statements with unterminated parentheses to continue to the next line.

`flake8` is the Python lint-checker. I recommend running it from time to time.

Reply:
Thanks for the suggestion!
I looked at your PR and took the changes from the setdefaults() section and did them in this PR, hopefully that makes merging simpler since I made a handful of changes in that section.